gulpfile.js

```diff
@@ -30,6 +30,12 @@ gulp.task('copy-vendors', function () {
 		.pipe(gulp.dest('./public/js'));
 });
 
+gulp.task('copy-assets', function () {
```

Comment:
Here's my refactor - I used `merge-stream` library to get async task using two `gulp.src` calls:

https://github.com/ScottMaclure/ractive-foundation/commit/d42af1a991052e5c7700e83aa9c1151f270e6f18